src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -536,9 +536,12 @@ export class LazyMobXTableStore<T> {
         } else {
             this.dataStore = new SimpleLazyMobXTableApplicationDataStore<T>(props.data || []);
         }
-
-        this.dataStore.page = 0;
-        this.itemsPerPage = props.initialItemsPerPage || 50;
+        if(this.dataStore.page === undefined) {
```

Comment:
@onursumer @kalletlak @adamabeshouse should we do this in contructor of SimpleLazyMobXTableApplicationDataStore?

Reply:
`initialItemsPerPage` is a parameter of `LazyMobXTable` component. I would prefer to leave as is for now.